images/linux/scripts/installers/validate-disk-space.sh

```diff
@@ -5,7 +5,7 @@
 ################################################################################
 
 availableSpaceMB=$(df / -hm | sed 1d | awk '{ print $4}')
-minimumFreeSpaceMB=18000
+minimumFreeSpaceMB=17500
```

Comment:
What is the accurate amount of free disk space will we have after your changes?
I a bit worried that we will affect customers

Reply:
1. success image - https://github.visualstudio.com/virtual-environments/_build/results?buildId=68046&view=results
    Available disk space: 17557 MB
2. failed image - https://github.visualstudio.com/virtual-environments/_build/results?buildId=68044&view=results
    Available disk space: 17557 MB
    Not enough disk space on the image (minimum available space: 17800 MB)